src/System.ValueTuple/pkg/System.ValueTuple.pkgproj

```diff
@@ -10,6 +10,8 @@
     <InboxOnTargetFramework Include="netcoreapp2.0" />
     <InboxOnTargetFramework Include="uap10.0.16299" />
     <InboxOnTargetFramework Include="$(AllXamarinFrameworks)" />
+
+    <HarvestIncludePaths Include="ref/portable-net40+sl4+win8+wp8;lib/portable-net40+sl4+win8+wp8" />
```

Comment:
@ericstj knows better but I think you want to add SupportedFramework=net45 here and bump the SupportedFramework to net461 in the ProjectReference for the ref above. I think that should get you a step closer to figuring out the validation error.